src/topotoolbox/utils.py

```diff
@@ -315,6 +322,141 @@ def get_cache_contents() -> (list[str] | None):
     print("Cache directory does not exist.")
     return None
 
+
+def read_from_cache(filename: str) -> GridObject:
+    """Read a GeoTIFF file from the cache directory and return it as a
+    GridObject. The filename should be the name of the GeoTIFF file. Find
+    available files by using 'get_cache_contents()'.
+
+    Parameters
+    ----------
+    filename : str
+        Name of the file to be read from the cache directory. Requires the
+        whole filename including the extension, like "dem.tif".
+    Returns
+    -------
+    GridObject
+        The GridObject generated from the cached GeoTIFF file.
+    """
+    cache_path = os.path.join(get_save_location(), f"{filename}")
+    grid_object = read_tif(cache_path)
+    return grid_object
+
+
+def load_open_topography(south: float, north: float, west: float, east: float,
+                         dem_type: str = "SRTMGL3", overwrite: bool = False,
+                         save_path: str | None = None,
+                         api_key: str = "demoapikeyot2021") -> GridObject:
+    """Download a DEM from Open Topography. The DEM is downloaded as a
+    GeoTIFF file and saved in the cache directory. The DEM is then
+    read into a GridObject. To overwrite an existing Downloaded DEM, use
+    the overwrite parameter. To save the DEM to a different location,
+    use the save_path parameter. The API key is required for accessing
+    the Open Topography API. The default API key is the demo key.
```

Comment:
We should also say something about reprojecting the DEM into a projected coordinate system here. See the docs for [readopentopo](https://github.com/TopoToolbox/topotoolbox3/blob/main/toolbox/IOtools/readopentopo.m). Also see below for the example.